ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/GDPR/SPGDPRNativePrivacyManagerViewController.swift

```diff
@@ -317,4 +322,32 @@ class FocusableScrollView: UIScrollView {
     }
 }
 
+/// A table view cell with a custom background configuration.
+/// It suppresses unwanted background and border effects that are automatically added by the system
+/// while while keeping the drop shadow and scaling effect when being focused.
```

Comment:
The second `while` can be safely removed

Reply:
✔️